site/docs/0.0/components/collapse.md

```diff
@@ -91,11 +89,9 @@ Using the [card]({{ site.baseurl }}/docs/{{ site.docs_version }}/components/card
   </div>
   <div class="card">
     <div class="card-header" id="headingTwo">
-      <h3 class="mb-0">
```

Comment:
I think we should add a text-size-h3 class

Reply:
Also, we should consider how screen readers and other semantics will be affected..

Reply:
mx-0 maybe?

Reply:
Adding back in `h3`s for the second example (follow-up items)